### ISSUE TITLE
Is it possible to expand a header by default?

### ISSUE THREAD

**Issue by iori57** [grennis/ExpandableRecyclerView#14]
When creating an ExpandableRecyclerAdapter.ListItem, is it possible to set a property so that it is expanded by default, instead of calling expandItems on each of them after finish with setItems()?

Thanks

**Comment by grennis** [comment]
That's a good enhancement idea. I have basically done the same in the past by calling expandItem immediately after setItems but it would be more efficient to populate it expanded in the first place. Will look into it more when I have time.

**Comment by iori57** [comment]
Cool, thanks for your time :)

Update: Even if I wanted to call expandItem manually after setItems I couldn't figure out how, because I need to expand multiple items, but after expanding the first one, I lost track of the index number needed to expand the other one.

Example:
Say I have 3 headers
Header 1
Header 2
Header 3

Previously user had expanded Header 1 and Header 2, so after reloading the data I would like to keep them expanded.
I call expandItem on index 0, Header 1 gets expanded..
now that there are a random amount of items under Header1, I couldn't figure out the index number needed to expand Header 2

at first I thought of implementing a way to retrieve index number of Header 2, but then found that it would be easier if we can configure it to be expanded by default, hence this idea :)

**Comment by grennis** [comment]
Yes you are correct - in my case I only had to expand one header, so I didn't hit this issue. It indicates there is a deeper problem that it is even possible to "lose track" of what items you are using. I will need to put some thought into it.

**Comment by grennis** [comment]
I had to implement this feature quickly in a pinch and didn't manage to get it working in the library, but was able to quickly make it work by re-expanding the items starting from the bottom, so:

```
for (int i=getItemCount()-1; i>=0; i--) {
    if (getItemViewType(i) == TYPE_HEADER && !isExpanded(i)) {
        ... expand this one if I want to ...
    }
}
```

This worked well enough as a workaround
